SPECS/ig/ig.spec

```diff
@@ -34,23 +34,35 @@ mkdir -p "%{buildroot}/%{_bindir}"
 install -D -m0755 bin/build/ig %{buildroot}/%{_bindir}
 
 %check
-make gadgets-unit-tests
-ig_file=$(mktemp /tmp/ig-XXXXXX.out)
-sudo ./bin/build/ig trace exec --host > $ig_file &
-ig_pid=$!
+set -e
+set -o pipefail
+
+# Inspektor Gadget provides unit tests but they rely on several components which
+# are not present in the chroot used to build and test the package, among
+# others:
+# * runc: https://github.com/inspektor-gadget/inspektor-gadget/blob/3c8d1455525b/pkg/container-hook/tracer.go#L302
+# * dockerd: https://github.com/inspektor-gadget/inspektor-gadget/blob/3c8d1455525b/pkg/container-utils/testutils/docker.go#L67
+# Even if we recreate a proper testing environment, we will still have problems
+# as, for example, the path tested will be inside the chroot while ig reports
+# the full path from host point of view.
+# For all these reasons, we will skip the unit tests and rather run a small
+# integration test.
+# Moreover, Inspektor Gadget CI covers Azure Linux extensively:
+# https://github.com/inspektor-gadget/inspektor-gadget/pull/1186/commits/066bf618d158
 sleep inf &
 sleep_pid=$!
-kill $ig_pid
+./bin/build/ig snapshot process --host | grep -qP "sleep\s+${sleep_pid}"
```

Comment:
Running locally it seems to break:
![image](https://github.com/microsoft/azurelinux/assets/4929861/1eb6b502-e4eb-4989-af9d-22eb51916a80)

@eiffel-fl I feel this is unexpected.

Reply:
I ran a test build on my build VM

Reply:
Ubuntu 22.x is my build VM.